src/opnsense/mvc/app/models/OPNsense/DHCP/DHCP.xml

```diff
@@ -0,0 +1,6 @@
+<model>
+    <mount>//OPNsense/DHCP</mount>
+    <description>DHCP configuration</description>
+    <version>1.0.1</version>
```

Comment:
Why 1.0.1? :)

Reply:
Ok I get it...this is a DHCPv4 model. Probably going to be confusing later on.. DHCP... DHCPv6 ... DHCPv4New ... DHCPv6New... we should be careful with the name as we can't reuse it later when we rewrite DHCP based on some other software.